releasenotes/README.md

```diff
@@ -26,12 +26,54 @@ Additional documentation:
 * [Salesforce CLI Setup Guide](https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_intro.htm)
 
 
-## 2.43.7 (May 29, 2024) [stable-rc]
+## 2.44.6 (Jun 5, 2024) [stable-rc]
 
 These changes are in the Salesforce CLI release candidate. We plan to include these changes in next week's official release. This list isn't final and is subject to change.
 
 ------------
 
+* NEW: (Beta) Enable a behavior of your project source files with the new `project convert source-behavior` command. For example, to update your project so it starts decomposing permission sets, run this command:
+
+    ```bash
+    $ sf project convert source-behavior --behavior decomposePermissionSetBeta
+    ```
+
+   When the command finishes, your `sfdx-project.json` file is updated to always decompose permission sets, and the existing permission set files in your local package directories are converted into the new decomposed format. You run this command only once for a given behavior change.  Here are the current possible values for the `--behavior` flag:
```

Comment:
If you wanted to, you could mention the `--dry-run` flag. Something like:
> Try the `--dry-run` flag to preview what would be changed. Sample files will be created in a `DRY-RUN-RESULTS` directory

Reply:
Updated the notes, thanks.